contracts/vaults/Withdrawals.sol

```diff
@@ -8,10 +8,11 @@ contract Withdrawals is Base {
 
     /**
     * @notice Submit a request to withdraw funds from the vault.
-    * The request will only be approved if the last action was a deposit or withdrawal or in case the last action was a withdraw request,
-    * that the pending period (of `generalParameters.withdrawRequestPendingPeriod`) had ended and the withdraw enable period (of `generalParameters.withdrawRequestEnablePeriod`)
-    * had also ended.
-    **/
+    * The request will only be approved if the last action was a deposit or
+    * withdrawal, or in case the last action was a withdraw request, that the
+    * withdraw request had expired (meaning that the pending period and the
+    * withdraw enable period had both ended).
```

Comment:
This comment doesn't really make sense to me, I'd completely re-write this

Reply:
I think we need a separate doc that describes the withdrawal logic, and then refer to that in the docstrings of withdrawRequest etc.

Reply:
i will make one, in the meantime see if this is clearer